packages/protocol/src/protocol.yml

```diff
@@ -3189,6 +3196,13 @@ Tracing:
       returns:
         traceName: string
 
+    tracingGroup:
+      parameters:
+        name: string
+        options: TracingGroupOptions
```

Comment:
i did think about future proof interface. many functions in Playwright nowadays go with options if they are optional and may extend in the future. 

I as an api user do like that, because those are named only and order is irrelevant and the chance breaking you code with an update is unlikely. if we for example want to support attachments to groups or a list of stack information or maybe a flag that the name contains HTML formatting, the options would govern us the freedom.

i was mostly aligning with test.step here.

but in the end your team decides.

Reply:
Or do you mean just in the communication interface between server and client? So only in the protocol?